main.css

```diff
@@ -27,22 +26,20 @@ h1 {
   color: black;
   position: relative;
   text-align: left;
-  margin: 20px;
-  left: 35px;
   text-decoration: none;
+  top: 30px;
 }
 h2 {
   text-align: center;
-  padding-top: 100px;
+  padding-top: 50px;
   text-decoration: underline;
 }
 div {
-  display: inline-block;
   border: solid;
   background-color: silver;
   width: 470px;
   text-align: justify;
-  margin: 2px;
+  margin: 4px;
   padding: 4px;
   border-radius: 20px;
 }
```

Comment:
For `section` you use `flex-direction: row;`, but `row` is a default value for `flex-direction` https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#Formal_definition